File: Barone.api/Models/CoccionModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace Barone.api.Models
{
    public class CoccionModel
    {
        [Key]
        public int id { get; set; }
        public RecetaModel Receta { get; set; }
        public DateTime Fecha { get; set; }
        public DateTime? FechaCoccion { get; set; }
        public DateTime? FechaFin { get; set; }
        public string NroLote { get; set; }
        public int Multiplicador { get; set; }
        public int Estado { get; set; }
        public FermentadorModel Fermentador { get; set; }
        public string MedicionesMash { get; set; }
        public string Hervor { get; set; }
        public string Lupulos { get; set; }
        public string Fermentacion { get; set; }
        public string Carbonatacion { get; set; }
        public string VolumenMediciones { get; set; }
        public string DetalleEmbarrilado { get; set; }
        public string DetalleEnlatado { get; set; }



    }
}

File: Barone.Client/src/app/modules/shared/models/coccion/coccion.model.ts
import { Constructable } from "src/app/core/models/constructable";
import { RecetaModel } from "../receta/receta.model";
import { FermentadorModel } from "../fermentador.model";
import { MashModel } from "./mash.model";
import { HervorModel } from "./hervor.model";
import { FermentacionModel } from "./fermentacion.model";
import { CarbonatacionModel } from "./carbonatacion.model";
import { LupuloModel, LupuloMediciones } from "../receta/lupulo.model";
import { VolumenModel } from "./volumen-medicion.model";

export class CoccionModel extends Constructable<Partial<CoccionModel>> {
    public id: number;
    public Receta: RecetaModel;
    public Fecha: number;
    public FechaFin: string;
    public FechaCoccion:string;
    public NroLote: string;
    public Multiplicador: number;
    public Estado: number;
    public Fermentador: FermentadorModel;
    public _medicionesMash: MashModel[];
    public _hervor: HervorModel[];
    public _lupulos: LupuloMediciones[];
    public _fermentacion: FermentacionModel;
    public _carbonatacion: CarbonatacionModel;
    public MedicionesMash: string;
    public Hervor: string;
    public Lupulos:string;
    public Fermentacion: string;
    public Carbonatacion: string;
    public VolumenMediciones:string;
    public _VolumenMediciones:VolumenModel[];
    public DetalleEmbarrilado:string;
    public _DetalleEmbarrilado:DetalleRendimiento;
}
export class DetalleRendimiento extends Constructable<Partial<DetalleRendimiento>>{
    public Latas: string;
    public Barriles:string;
    public TotalLitros: string;
    public DiasEnTanque:string;
}